src/components/Autocomplete/Autocomplete.tsx

```diff
@@ -69,8 +73,15 @@ export class Autocomplete extends React.Component<Props> {
    * TODO: Benchmark - do we need a faster comparison here?
    */
   shouldItemRender = (item: string, value: string) =>
-    (this.props.pinnedItems && this.props.pinnedItems.includes(item))
-    || item.toLowerCase().includes(value.toLowerCase())
+
+    // If the default value is selected, show all items
```

Comment:
. at the end please :)

Reply:
I am more than happy to use an automatic lint rule if you want to add it :)